chain-db-watcher/src/index.ts

```diff
@@ -147,47 +155,60 @@ async function main (): Promise<void> {
 		},
 		err => {
 			console.error(chalk.red(err));
+		},
+		() => {
+			console.log('Proposal subscription completed');
+			process.exit(1);
 		}
 	);
 
-	referendumSubscriptionClient.subscribe(({ data }): void => {
-		if (data?.referendum.mutation === subscriptionMutation.Created) {
-			const {
-				preimageHash,
-				referendumId,
-				referendumStatus
-			} = data?.referendum?.node;
-
-			// At referendum creation, there should be only
-			// a "Started" status event.
-			if (!(referendumStatus?.[0]?.status === eventStatus.Started)) {
-				console.error(
-					chalk.red(
-						`Referendem with id ${referendumId.toString()} has an unexpected status. Expect "${eventStatus.Started}", got ${referendumStatus?.[0]?.status}."`
-					)
-				);
-				return;
-			}
-
-			if (!preimageHash) {
-				throw new Error(`Unexpect preimage hash, got ${preimageHash}`);
-			}
-
-			if (!referendumId && referendumId !== 0) {
-				throw new Error(`Unexpect referendumId, got ${referendumId}`);
+	referendumSubscriptionClient.subscribe(
```

Comment:
The changes look big here, but I just changed the indentation and added the `complete` callback as I did with others.